docs/interceptors.md

```diff
@@ -242,24 +246,26 @@ spec:
 
 ### Bitbucket `Interceptors`
 
-A Bitbucket `Interceptor` contains logic that validates and filters [Bitbucket server](https://confluence.atlassian.com/bitbucketserver) webhooks.
-It can validate the webhook's origin as described in [Webhooks](https://docs.gitlab.com/ee/user/project/integrations/webhooks.html). It does
-not support [Bitbucket cloud](https://support.atlassian.com/bitbucket-cloud/) due to lack of support for validating secrets; if you need this
-functionality with Bitbucket, use a [CEL `Interceptor`](#cel-interceptors) instead. The Bitbucket `Interceptor` always preserves  the payload
+Bitbucket interceptors has support for both Bitbucket server and Bitbucket cloud.
+
+### Bitbucket Server
+
+A Bitbucket Server `Interceptor` contains logic that validates and filters [Bitbucket server](https://confluence.atlassian.com/bitbucketserver) webhooks.
```

Comment:
Thanks @savitaashture I think separating out the difference between server and cloud is great. One minor thing is that  from the docs it seems like we have two separate interceptors. We only have one bitbucket interceptor which supports secret validation and event filtering for bitbucket server and only event filtering for bitbucket cloud

Reply:
Ye looks like that
Updated now
PTAL